exercises/binary-search/instructions.md

```diff
@@ -5,17 +5,17 @@ Your task is to implement a binary search algorithm.
 A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
 It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
 
-```exercism/caution
+````exercism/caution
 Binary search only works when a list has been sorted.
-```
+````
 
 The algorithm looks like this:
 
-- Divide the sorted list in half and compare the middle element with the item we're looking for.
-- If the middle element is our item, then we're done.
-- If the middle element is greater than our item, we can eliminate that number and all the numbers **after** it.
-- If the middle element is less than our item, we can eliminate that number and all the numbers **before** it.
-- Repeat the process on the part of the list that we kept.
+- Find the middle element of a sorted list and compare it with the item we're looking for.
+- If the middle element is our item, then we're done: return its index.
+- If the middle element is greater than our item, we can eliminate that element and all the elements **after** it.
+- If the middle element is less than our item, we can eliminate that element and all the elements **before** it.
+- Repeat the process on the part of the list that has not been eliminated. If all the list has been eliminated then the item is not in the list.
```

Comment:
Split this into two bullet points?

```suggestion
- If list entirely list has been eliminated then the item is not in the list.
- Repeat the process on the part of the list that has not been eliminated. 
```

Reply:
Ah, I like that.

Reply:
How about this?

```suggestion
- If every element of the list has been eliminated then the item is not in the list.
- Otherwise, repeat the process on the part of the list that has not been eliminated.
```

Reply:
Yeah, that's nice, @cmcaine. It  flows even better.